.github/actions/spelling/expect/alphabet.txt

```diff
@@ -1,20 +1,15 @@
-AAAAA
-AAAAAAAAAAAAA
```

Comment:
Most of the files in this PR will either be deletions or minor modifications. I recommend just ticking them off as "viewed" first. Afterwards it'll still be a large PR, but it'll be manageable I think.